vNext/extensions/applicationinsights-react-native/Tests/TestFramework/TestClass.ts

```diff
@@ -0,0 +1,250 @@
+/// <reference path="../External/sinon.d.ts" />
+/// <reference path="../External/qunit.d.ts" />
+/// <reference path="Assert.ts" />
+/// <reference path="./TestCase.ts"/>
+
+class TestClass {
```

Comment:
we could possibly create a test package in future that has all these files and share that across all these individual packages. Seems like a good opportunity to clean up.